format/spec.md

```diff
@@ -631,6 +632,29 @@ When expiring snapshots, retention policies in table and snapshot references are
     2. The snapshot is not one of the first `min-snapshots-to-keep` in the branch (including the branch's referenced snapshot)
 5. Expire any snapshot not in the set of snapshots to retain.
 
+#### Statistics file
+
+Statistics files are valid [Puffin files](../puffin-spec). Statistics are informational. A reader can choose to
+ignore statistics information. Statistics support is not required to read the table correctly.
+
+Statistics files' metadata within `statistics` table snapshot field is a struct with the following fields:
+
+| v2         | Field name                      | Type                              | Description                                                                                                                               |
+|------------|---------------------------------|-----------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------|
+| _required_ | **`statistics-path`**           | `string`                          | Path of the statistics file. See [Puffin file format](../puffin-spec).                                                                    |
+| _required_ | **`file-size-in-bytes`**        | `long`                            | Size of the statistics file.                                                                                                              |
+| _required_ | **`file-footer-size-in-bytes`** | `long`                            | Total size of the statistics file's footer (not the footer payload size). See [Puffin file format](../puffin-spec) for footer definition. |
+| _required_ | **`source-sequence-number`**    | `long`                            | Table sequence number at which the stats were calculated                                                                                  |
```

Comment:
curious to know, what's the recommended frequency to run stats writing application ? will not the stats file become outdated after a new commit on the same partition?

Reply:
Stats are supposed to be approximate information about the data, so they don't need to be exact to still be useful.
When to use stats and when not to -- it's supposed to be up to the query engine to decide.

For writing stats with Trino, there is a PR: https://github.com/trinodb/trino/pull/12317.